web-api/migration-terraform/bin/deploy-app.sh

```diff
@@ -2,6 +2,10 @@
 
 ENVIRONMENT=$1
 
+[ -z "${ENVIRONMENT}" ] && echo "You must pass in ENVIRONMENT as a command line argument 1" && exit 1
+[ -z "${SOURCE_TABLE}" ] && echo "You set SOURCE_TABLE as an environment variable" && exit 1
```

Comment:
where is SOURCE_TABLE and DESTINATION_TABLE being used?

Reply:
They're part of the blue / green migrations.